src/bin.js

```diff
@@ -22,6 +22,7 @@ program
   .option("-b, --browsers <items>", "browsers list (comma separated)")
   .option("-I, --no-import", "do not inline @import")
   .option("-U, --no-url", "do not adjust url()")
+  .option("-e, --extract", "output values of custom properties")
```

Comment:
custom media are not variables. I will use "extract some resolved values in a file (custom properties + custom media)". For now we have only those and if too many came in, we will need to write that in the doc, somewhere else than just in this minimal help.